extensions/EXT/EXT_texture_sRGB_decode.txt

```diff
@@ -221,26 +223,33 @@ Additions to Chapter 3 of the 2.1 Specification (Rasterization)
     Add after the first paragraph of the section:
 
    "The conversion of sRGB color space components to linear color space is
-    always performed if the texel lookup function is one of the texelFetch
-    builtin functions.
+    always applied if the TEXTURE_SRGB_DECODE_EXT parameter is DECODE_EXT.
+    Table X.1 describes whether the conversion is skipped if the
+    TEXTURE_SRGB_DECODE_EXT parameter is SKIP_DECODE_EXT, depending on
+    the function used for the access, whether the access occurs through a
+    bindless sampler, and whether the texture is statically accessed
+    elsewhere with a texelFetch function."
 
-    Otherwise, if the texel lookup function is one of the texture builtin
-    functions or one of the texture gather functions, the conversion of sRGB
-    color space components to linear color space is controlled by the
-    TEXTURE_SRGB_DECODE_EXT parameter.
+    Add a new table X.1, Whether the conversion of sRGB color space
+    components to linear color space is skipped when the
+    TEXTURE_SRGB_DECODE_EXT parameter is SKIP_DECODE_EXT.
 
```

Comment:
When initially looking at this, I found the new table a little unclear.  After staring at this for a bit, i'm ok with it as-is.

Reply:
+1 I've had a look at this as well and the changes seem good to me.